modepy/tools.py

```diff
@@ -397,6 +400,42 @@ def try_add_tet(d1, d2, d3, d4):
 submesh = MovedFunctionDeprecationWrapper(simplex_submesh)
 
 
+def hypercube_submesh(node_tuples):
```

Comment:
Shamelessly copied from [meshmode](https://github.com/inducer/meshmode/blob/master/meshmode/mesh/tools.py#L54).

Reply:
Started bumping versions in eaa8500.

What do you think of the rename? Is it ok?

Reply:
Rename is fine.